How to Build Your Own Website - Introduction to HTML, CSS, and Javascript

emily-gui opened the pull request:
### Reference to issue
- #33 

### Description of the changes proposed in the pull request
- completed blog 
- everything inside the folder (how to build your own website)

### Reviewers requested:
- @atuljayaram 
- @your_PM_github_username
- @your_QA_github_username

atuljayaram commented on how to build your own website/blog.md:
```diff
@@ -0,0 +1,253 @@
+
+### Building the Foundation: HTML File
+
+*HTML* stands for Hypertext Markup Language and is the standard markup language for displaying content on a website. 
```
Maybe briefly describe what a "markup language" is.

atuljayaram commented on how to build your own website/blog.md:
```diff
@@ -0,0 +1,253 @@
+
+If the text you wanted to add to the page has line breaks in between, you must up another `<p></p>` element because HTML will not change be able to detect and display the indents in your code onto your webpage.
+
+To add an image to your web page, we will use an image tag `<img>` which does not have a closing tag because it is a *void element*. The `<img>` tag will include *attributes*, which can be added to the heading tag of an element to provide additional information, and the two attributes we will use as `src` and `alt`. The `src` attribute will contain the URL or name of the image, and in case the im ...
```
Briefly elaborate on what a void element is.

atuljayaram commented on how to build your own website/blog.md:
```diff
@@ -0,0 +1,253 @@
+# How to Build Your Own Website - Introduction to HTML, CSS, and JavaScript
+
+The world of web development and design is growing, and creating a website, whether it would be to display a design portfolio or to track personal progress in everyday life, has become a popular method to establish an online presence or to track personal progress. As the selection of website builders and frameworks continue to expand, however, it can seem daunting and overwhelming when you do not ...
```
You repeat yourself here a bit with the "personal progress" phrase. Please go through the entire blog and thoroughly proofread it, to catch any awkward wording or grammar mistakes!

emily-gui commented:
Thank you for the feedback! I just pushed my revisions, and if there is anything else you would like me to change, feel free to let me know.

emily-gui commented:
For the editor's reference, I gained inspiration these two blogs:
- https://docs.microsoft.com/en-us/learn/modules/build-simple-website/
- https://workshops.hackclub.com/personal_website/

I included these links in my blog choice request, but I thought it would be important to include them here, in case the editor thinks a part of my blog is too similar to these blogs, to avoid the possibility of plagiarism.

vkxu657 commented on how to build your own website/blog.md:
```diff
@@ -0,0 +1,254 @@
+If you want to enter multiple blocks of text separated by line breaks, you must use another `<p></p>` element for each indented block of text. HTML will not be able to detect or display the line breaks in your code onto your web page.
+
+To add an image to our web page, we will use an image tag `<img>`, which does not have a closing tag because it is a *void element*. Void elements can't hold any kind of content besides *attributes*, which are additional bits of information featured in an element's heading tag. The `<img>` tag will include two attributes, `src` and `alt`. The `src` attribute will contain the URL or file name of t ...
+
```
The HTML code or image here doesn't show up in the blog post preview.

vkxu657 commented:
btw @emily-gui if you referenced both of those sources (the tutorials from Microsoft and Hackclub) in a way that is particular to what they wrote (i.e., whatever you may have used is past the point of general knowledge and can only be found from those sources), I'd cite them in the post via footnotes (at the end) and their corresponding superscripts (wherever the info you found is being used). Otherwise, since HTML is so universal and this tutorial isn't particularly tied to just one source, I wouldn't worry about citing b/c it's essentially common knowledge. 
If you're ever unsure, this list of when to cite sources is really helpful: https://poorvucenter.yale.edu/writing/using-sources/understanding-and-avoiding-plagiarism/warning-when-you-must-cite#2.%20When%20you%20introduce

emily-gui commented:
Okay, thank you so much for looking over my blog! I will make sure to edit my blog to your requests by this weekend! After I finish fixing my blog, should I ask you to look over it again, or is there someone else I should notify?
@vkxu657

vkxu657 commented:
@emily-gui Tag me and I'll look it over once you're done!

emily-gui commented:
@vkxu657 Okay, will do! I'll have it done this weekend!

emily-gui commented:
@vkxu657 I finished editing my blog! I decided that the best way to cite the JavaScript code was to write it in the text itself, because if I added it in the comments of the code blocks, it may confuse viewers that are beginners at coding since they did not need to include that comment when they are creating the websites themselves. So, I added the link to the webpage I based the code from at the end of the first paragraph of the "Adding Interactivity with JavaScript" section. Thanks for reviewing my blog, and please let me know if there is anything else I need to do!
